CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 All notable changes to this project are documented in this file.
 
+## 0.27.0
+
+**Release date:** 2022-08-08
+
+This prerelease comes with support for the `OCIRepository` source type.
+
+In addition, the controller has been updated to Kubernetes v1.24.3
+and Kustomize v4.5.7.
+
```

Comment:
No list of the changes?

Reply:
Forgot to push them, check it now please.